src/patternfly/components/Check/styles.scss

```diff
@@ -1,9 +1,23 @@
 @import "../../patternfly-utilities";
 
 .pf-c-check {
-  cursor: pointer;
+  --pf-c-check__label--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-check__label--diabled--color: var(--pf-global--disabled-color--100);
 
-  &:disabled {
+  .pf-c-check__label.pf-m-disabled, .pf-c-check__input:disabled {
+    color: var(--pf-c-check__label--diabled--color);
     cursor: not-allowed;
   }
+
+  .pf-c-check__input, .pf-c-check__label {
```

Comment:
can you put `.pf-c-check__label` on it's own line? I'll look to see if I can create a stylelint rule for this.